packages/functional-tests/pages/login.ts

```diff
@@ -16,18 +16,19 @@ export const selectors = {
   ERROR: '.error',
   LINK_LOST_RECOVERY_KEY: '.lost-recovery-key',
   LINK_RESET_PASSWORD: 'a[href^="/reset_password"]',
-  LINK_USE_DIFFERENT: '#use-different',
+  LINK_CHANGE_EMAIL: 'a:has-text("Change email")',
+  LINK_USE_DIFFERENT: 'a:has-text("Use a different account")',
   LINK_USE_RECOVERY_CODE: '#use-recovery-code-link',
   NUMBER_INPUT: 'input[type=number]',
-  PASSWORD: '#password',
-  PASSWORD_HEADER: '#fxa-signin-password-header',
+  PASSWORD: ':nth-match(input[type=password],1)',
```

Comment:
Note to self: let's update these when we remove Backbone Signup or work on signin. I know it's tricky if the DOM is different 😓

Reply:
Added a comment/TODO